examples/cryptocurrency-advanced/README.md

```diff
@@ -117,6 +118,19 @@ exonum-cryptocurrency-advanced run --node-config example/3/node.toml --db-path e
 exonum-cryptocurrency-advanced run --node-config example/4/node.toml --db-path example/4/db --public-api-address 0.0.0.0:8203 --master-key-pass pass
 ```
 
+Before service deploy make sure that you have pure python implementation of protobuf:
```

Comment:
Interesting. @popzxc what is the reason? Shouldn't it be specified in the `exonum_launcher`?

Reply:
It has been specified in exonum-python-client.

Reply:
I see now. Unfortunate problem :(

Reply:
>what is the reason?

The reason is extremely "good" protobuf implementation for python: it ignores the `package` field set in files and thinks that two files with the same name are always belong to the same descriptor (and we have plenty of those, e.g. `service.proto`). The intended way (I asked about it in the protobuf repo) is to compile files with the same name providing the folders name as well, so they eventually will become different names.

Pure python implementation softens those restrictions, but they've said that they want to implement this behaviour for pure python library as well :cold_sweat: 

I'm currently thinking what can we do about it, but for now using the pure python implementation is the simplest approach we can get.